ffs/scheduler/scheduler.go

```diff
@@ -422,7 +428,29 @@ func (s *Scheduler) executeQueuedStorage(j ffs.StorageJob) {
 
 	// Execute
 	s.l.Log(ctx, "Executing job %s...", j.ID)
-	info, dealErrors, err := s.executeStorage(ctx, a, j)
+	dealUpdates := make(chan ffs.FilStorage)
+	go func() {
+		for update := range dealUpdates {
```

Comment:
Handle deal updates by updating our in memory cache, and doing some debug logging for now.